ocpp_codec/v20/messages.py

```diff
@@ -117,6 +117,15 @@ class Response:
         setVariableResult: typing.List[types.SetVariableResultType]
 
 
+class StatusNotification(Action):
+    @dataclass
+    class Request:
```

Comment:
Why only add `Request`?
If we add a new message we should add the _Request_ and the _Response_, both will be needed at one point.

Reply:
`Action` already defines a `Response` that `StatusNotification` inherits and the specification reads `No fields are defined.` for `StatusNotificationResponse`.

Reply:
For various reasons (logging, comparison), I ended up overriding `Response` with an empty method just to have it defined.

Reply:
Yes this is the right way to go with the current implementation. The `Action.Response` is to be taken as some sort of example, we should rework this a bit.